templates/controller-validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.controller.enabled }}
```

Comment:
I think we can delete this file now. We only need the one webhook config as we will probably do validation and defaulting using the mutating webhook